packages/node-sdk/paima-db/migrations/up.sql

```diff
@@ -98,6 +98,59 @@ CREATE TABLE delegations (
  PRIMARY KEY (from_id, to_id)
 );
 
+-- Create a function to notify any change in address or delegate tables
+create or replace function public.notify_wallet_connect()
```

Comment:
Although I think this is not bad to avoid blocking on this, but I can't image this scaling well if we need to cache more things in the future like the achievement progress of users since this is a lot of code and complexity per cache invalidation logic

Reply:
This a generic function, it transforms any trigger into a nice json, with table / method / data 
```
create or replace function public.notify_wallet_connect()
returns trigger
  language plpgsql
as $function$ 
begin
   perform pg_notify('wallet_connect_change', payload);
    return null;
end;
$function$
;
``` 

Is enough, but does not differentiate between table/method or give info about changed data